trackier-ios-sdk/Classes/TrackierSDKInstance.swift

```diff
@@ -166,5 +174,13 @@ class TrackierSDKInstance {
         }
     }
     
+    func deviceTokenApns() {
+        let wrk = makeWorkRequest(kind: TrackierWorkRequest.KIND_Token)
+        wrk.deviceToken = deviceToken
```

Comment:
only send request if token has changed or token is not stored in the local cache (first time request)